src/jumio-kyc/kyc.controller.spec.ts

```diff
@@ -392,30 +392,30 @@ describe('KycController', () => {
       expect(body).toMatchObject({
         data: [
           {
-            airdrop_completed_by: '2023-03-16T00:00:00.000Z',
+            airdrop_completed_by: '2023-04-22T00:00:00.000Z',
```

Comment:
This is 22 instead of 21, where the previous version had the same date (16 and 16) -- just double checking this is intentional

Reply:
no accident, test failing 👯

Reply:
Didn't add updates before last force push